policies/mandatory-file-license.yml

```diff
@@ -0,0 +1,36 @@
+# metadata
+name: This repo is missing a license file
+description: This is a config to check if LICENSE.MD is present in repo.
+
+# filters
+resource: repository
+where:
+- |
+  repository.name.startsWith("asdfasdfasdfasdfadsf", StringComparison.InvariantCultureIgnoreCase) 
```

Comment:
Can you remove this line, it was a dummy line we added. and after that also remove "||" from next line